bash_scripts/Proj.sh

```diff
@@ -18,13 +18,13 @@ case $1 in
         # We occasionally get Address already in use errors likely because of cron runing jobs in parallel.
         #    Here we will just use a different redirect uri to deconflict them
         export SPOTIPY_REDIRECT_URI='http://localhost:9091'
```

Comment:
bro i think u spelled this wrong

Reply:
wait hold up what is spotipy

Reply:
you weren't lying huh? However, that's actually how it's spelled. 'spotipy' is the api wrapper I am currently using for Spotify.

Reply:
technically, this works